liquidity_pool_router/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 publish = false
 
 [lib]
-crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib"]
 
 [features]
 testutils = ["soroban-sdk/testutils", "soroban-auth/testutils","soroban-liquidity-pool-contract/testutils","dep:ed25519-dalek", "dep:sha2", "dep:stellar-xdr"]
```

Comment:
I'm surprised this builds and runs. Under some conditions I've seen crates fail to import other crates if they were only a cdylib and not a rlib. It looks like this crate is importing `soroban-liquidity-pool-contract`, and that crate has no rlib.

Reply:
Ah actually this contract only uses `soroban-liquidity-pool-contract` through the wasm file. All references to it in `Cargo.toml` can be removed.